apps/framework-cli/src/cli/settings.rs

```diff
@@ -57,6 +58,7 @@ impl Default for Telemetry {
 pub struct Settings {
     #[serde(default)]
     pub logger: LoggerSettings,
+    #[allow(dead_code)]
     #[serde(default)]
     pub features: Features,
```

Comment:
Do we need this for anything?

Reply:
Not anymore - we could reuse but it is not currently used